### ISSUE TITLE
Match is not defined

### ISSUE THREAD

**Issue by unique-vv** [meteor-vue/meteor-integration#4]
```
if (args.length) {
      const lastParam = args[args.length - 1];

      // Match pattern to check if the last arg is an options argument.
      const lastParamOptionsPattern = {
        onReady: Match.Optional(Function),
        onStop: Match.Optional(Function),
        connection: Match.Optional(Match.Any),
      };

      if (typeof lastParam === 'function') {
        options.onReady = args.pop();
      }
      else if (lastParam && !isEmpty(lastParam) && Match.test(lastParam, lastParamOptionsPattern)) {
        options = args.pop();
      }
    }
```
in your /install.js line 72

**Comment by mitar** [comment]
Nice catch.

**Comment by mitar** [comment]
I was developing with those packages being added globally, so I missed those.

I released 0.1.1 version of this package. Please test.
